scripts/build_release.sh

```diff
@@ -58,7 +41,13 @@ export VARIATIONS="base"
 export NO_BUILD=true
 if [ -z "${RSTAMP}" ]; then
     RSTAMP=$(scripts/reverse_hex_timestamp)
+    echo RSTAMP=${RSTAMP} > "${HOME}/rstamp"
 fi
+# What's my default IP address?
+# get the datacenter IP address for this EC2 host.
+# this might equivalently be gotten from `netstat -rn` and `ifconfig -a`
+DC_IP=$(curl --silent http://169.254.169.254/latest/meta-data/local-ipv4)
```

Comment:
Can we error out if this returns "" - assuming this script should only be run on an EC2 instance?

Reply:
updated to allow external setting, try to fetch like this otherwise, and fail if ultimately empty